libs/ui/button/helm/src/lib/hlm-button.directive.ts

```diff
@@ -33,31 +34,34 @@ export type ButtonVariants = VariantProps<typeof buttonVariants>;
 @Directive({
 	selector: '[hlmBtn]',
 	standalone: true,
+	exportAs: 'hlmBtn',
 	host: {
-		'[class]': '_computedClass()',
+		'[class]': 'computedClass()',
 	},
 })
 export class HlmButtonDirective {
-	public readonly userClass = input<ClassValue>('', { alias: 'class' });
-	private readonly _settableClass = signal<ClassValue>('');
+	/** Allow the default values to be overridden. */
+	private readonly _config = injectBrnButtonConfig();
 
-	protected _computedClass = computed(() =>
-		hlm(buttonVariants({ variant: this._variant(), size: this._size() }), this._settableClass(), this.userClass()),
+	/** Allow additional classes to be defined. */
```

Comment:
Do the comments add value here or is the code/variable naming self explanatory enough?

Reply:
I can remove these